cmd/derper/derper.go

```diff
@@ -235,24 +235,26 @@ func main() {
 			cert.Certificate = append(cert.Certificate, s.MetaCert())
 			return cert, nil
 		}
-		go func() {
-			port80srv := &http.Server{
-				Addr:        net.JoinHostPort(listenHost, "80"),
-				Handler:     certManager.HTTPHandler(tsweb.Port80Handler{Main: mux}),
-				ReadTimeout: 30 * time.Second,
-				// Crank up WriteTimeout a bit more than usually
-				// necessary just so we can do long CPU profiles
-				// and not hit net/http/pprof's "profile
-				// duration exceeds server's WriteTimeout".
-				WriteTimeout: 5 * time.Minute,
-			}
-			err := port80srv.ListenAndServe()
-			if err != nil {
-				if err != http.ErrServerClosed {
-					log.Fatal(err)
+		if (*certMode == "letsencrypt") {
```

Comment:
don't need these parens.

But let's do this more explicitly with a new flag, rather than tying it to the TLS mode.

It has nothing to do with LetsEncrypt.

Reply:
@bradfitz I've updated the PR with a more comprehensive changeset following your comment. Let me know